repository/src/main/resources/alfresco/application-context-highlevel.xml

```diff
@@ -44,4 +44,5 @@
     <import resource="classpath*:alfresco/deprecated-model-bootstrap-context.xml" />
     <import resource="classpath*:alfresco/events-context.xml" />
     <import resource="classpath*:alfresco/messaging-context.xml" />
+    <import resource="classpath:alfresco/invitation-service-context.xml"/>

```

Comment:
What is the reason of moving this context file here?

Reply:
This fixes the minimal context run test. this bean is having a couple of dependencies that are not required for the minimal run. so I moved the inviting service here.